.github/workflows/test.yml

```diff
@@ -10,8 +10,6 @@ on:
     branches:
       - 'master'
       - '[0-9]+.[0-9]+'
-    tags:
-      - 'v*'
```

Comment:
Does this mean we don't have a "check" on the tag itself? (I guess no real loss, as we'd have checks on the commit the was tagged, but I guess it's nice to have a more discoverable check for the tag))

Reply:
Yes I was thinking it was not necessary as we are tagging on a commit hat was already checked anyway